packages/office-ui-fabric-react/src/components/Persona/Persona.types.ts

```diff
@@ -206,6 +217,10 @@ export interface IPersonaStyles {
   textContent: IStyle;
 }
 
+export interface IPersonaCoin {
+  focus: () => void;
```

Comment:
focus should be reserved for interactive elements.

Reply:
👍